index.html

```diff
@@ -71,30 +76,31 @@ <h3>See Project</h3>
                 </li>
             </ul>
         </div>
-        <div class="projects">
-            <img class="img" src="./images/Snapshoot Portfolio (1).png">
+        <div class="projects reverse">
+            <img class="img" src="./images/Snapshoot Portfolio 2.png">
 
             </img>
             <ul class="projectInfo">
                 <li class="mps">
-                    <h3>Multi-Post <br> Stories</h3>
+                    <h3>Uber Navigation</h3>
                 </li>
                 <li class="techstack">
                     <div>
                         <ul>
-                            <li class="canopy">CANOPY</li>
-                            <li class="bed">Back End Dev</li>
-                            <li class="year">2015</li>
+                            <li class="canopy">UBER</li>
+                            <li class="bed">Lead Developer</li>
+                            <li class="year">2018</li>
                         </ul>
                     </div>
                 </li>
                 <li class="aboutProject">
-                    A daily selection of privately <br> personalized reads: no accounts or <br> sign-ups required.
-                </li>
+                    A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive
+                    computer: your car. </li>
```

Comment:
- [OPTIONAL]  This is supposed to be a paragraph. `li` tags are used to present items that are part of a list or navigation menu. What you have done here is semantically incorrect and I suggest that you change it.